app/Http/Controllers/TodoController.php

```diff
@@ -36,9 +41,20 @@ public function show(Todo $todo)
         // Show a single Todo
     }
 
-    public function update(Request $request, Todo $todo)
+    public function update(TodoUpdateRequest $request, String $id): JsonResponse
```

Comment:
Use route model binding, so you can pass the `Todo` model directly as an argument:

https://laravel.com/docs/10.x/routing#route-model-binding